packages/core/src/cli/index.ts

```diff
@@ -49,14 +49,11 @@ const argv = yargs
     description: `Show what would be built (or deleted, if specified with '--clean'). Same as the TS \`--dry\` flag.`,
     type: 'boolean',
   })
-  // TODO: implement these!
```

Comment:
🎉

Reply:
Just need to do `--watch` now. And spoilers: that's just a matter of exercising the functionality thoroughly with the rest suite but it's working as expected with every test I convert, no changes required!